index.html

```diff
@@ -1,28 +1,29 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
   <title>try.hood.ie</title>
   <link rel="stylesheet" type="text/css" href="css/main.css">
   <link href='http://fonts.googleapis.com/css?family=Roboto+Slab:400,300' rel='stylesheet' type='text/css'>
 </head>
 <body>
-<img src="img/stripe.gif" id="stripe" />
-<header> 
+<img src="img/stripe.gif" id="stripe" alt=""/>
+<header role="banner"> 
     <h1>try.hood.ie</h1>
     <h2>create your app in a finger snap</h2>
-  </header>
-<div class="container">
+</header>
+<div role="main" class="container">
   <section class="current">
   		<form action="create">
         <div class="">
-          <input name="subdomain" required class="left" placeholder="your appname" />
+          <label for="subdomain"><b>Choose an application name</b></label><br />
+          <input id="subdomain" name="subdomain" required class="left" placeholder="your appname" />
           <span class="left label">.hoodieapp.com</span>
         </div>
   			<button type="submit" class="btn right hid">create me</button>
   		</form>
   </section>
 
-  <section class="desc">
+  <section class="desc" tabindex="-1" id="congratulations">
```

Comment:
tabindex is basically the order of things when you tab through the site using only a keyboard. Setting it to -1 removes this from the tab index so that you have better control over the order of how things are read out by voiceover or other assistive technologies. :) @Charlotteis can comment more because she'll have done the testing to figure out the tab order.